client/dashboard/profile-wizard/steps/theme/error.js

```diff
@@ -0,0 +1,31 @@
+/** @format */
+
+export default () => (
+	<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think we can use the `info` Gridicon here.

Reply:
Nice find!  I think this is close enough to the original icon.  @jameskoster Let me know if you see any issues with using this one instead. (part of the gridicon library we're already using)

<img width="282" alt="Screen Shot 2019-06-26 at 11 06 12 AM" src="https://user-images.githubusercontent.com/10561050/60148822-2684b300-9805-11e9-9e30-69b207a69094.png">